pkg/apis/mattermost/v1alpha1/clusterinstallation_utils.go

```diff
@@ -230,8 +230,43 @@ func (mattermost *ClusterInstallation) GenerateDeployment(dbUser, dbPassword str
 		},
 	})
 
-	// Generate Minio config
 	minioName := fmt.Sprintf("%s-minio", mattermost.Name)
+	// Create the init container to create the MinIO bucker
+	initContainers = append(initContainers, corev1.Container{
+		Name:            "create-minio-bucket",
+		Image:           "minio/mc:latest",
+		ImagePullPolicy: corev1.PullIfNotPresent,
+		Command: []string{
+			"/bin/sh", "-c",
+			fmt.Sprintf("mc config host add localminio http://%s $(MINIO_ACCESS_KEY) $(MINIO_SECRET_KEY) && mc mb localminio/%s -q -p", minioService, mattermost.Name),
+		},
+		Env: []corev1.EnvVar{
+			{
+				Name: "MINIO_ACCESS_KEY",
+				ValueFrom: &corev1.EnvVarSource{
```

Comment:
Can we turn these `*corev1.EnvVarSource` into variables and reuse them for the env vars on the app server too?

Reply:
yes

Reply:
done @jwilander